recipe/meta.yaml

```diff
@@ -10,40 +10,43 @@ source:
   sha256: 4bb1cce559418f1a9c0c366c05d8b62161f4a3ed7f6283f4bb3e934db09e20b1
 
 build:
-  number: 0
+  number: 1
   noarch: python
   script: "{{ PYTHON }} -m pip install . -vv"
 
 requirements:
   host:
-    - python >=3.5
+    - python >=3.6
     - setuptools
     - pip
     - pytest-runner
   run:
-    - python >=3.5
+    - python >=3.6
     - scipy >=1.2
     - numpy >=1.15
     - pandas >=0.23
     - fiona >=1.8
     - rtree >=0.8
     - geojson >=2.5.0
     - shapely >=1.6
-    - geopandas >=0.4
+    - geopandas >=0.6
     - cftime >=1.0.3.4,<1.0.4
     - netCDF4 >=1.4
     - dask >=1.2
     - bottleneck >=1.2.1,<1.3
-    - xarray ==0.13.0
+    - xarray ==0.14.1
     - pyproj ==2.4.1
-    - pint >=0.8
+    - pint >0.8,<0.10
     - boltons >=19.0
     - scikit-learn >=0.21.3
     - pytest
 
 test:
   imports:
     - xclim
+    - xclim.atmos
```

Comment:
xclim.land
xclim.seaIce

Reply:
Scrap that, importing xclim now imports atmos, land and seaIce. So you may remove all of them.

Reply:
Is this true for 0.12? I can revert this in the 0.13 build tomorrow.

Reply:
No, you're right.